### PR TITLE
Feature python database integration

### PR DESCRIPTION
**Pull request by upendras-ltts**
Setup the python environment in system and IDE
Added SQLite database to store the Todo data.
Added the model class relevant fields to match with existing server code.
Created the repository function to store and retrieve the Todo data.
Integrated Flask to route the Api's for CRUD operation.

### CODE REVIEW

File: pythontodo/src/todo.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        self.task = task
+        self.description = description
+        self.completed = completed
+        self.edited = edited
+        self.createdDate=createdDate
+        self.modifiedDate=modifiedDate
+        self.lastViewed = lastViewed
+        self.serialNumber = serialNumber
+        
+        engine = create_engine('sqlite:///todos.db')
```
Comment (daniel-statsig) [inline pythontodo/src/todo.py:34]: we don't need a Database, just do it in memory. Using a database adds complexity that is not needed.

File: pythontodo/src/todo_controller.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+@app.route('/todos', methods=['GET'])
+def get_all_todos():
+    todos = repository.get_all_todos()
+    return jsonify([todo.getJson() for todo in todos])
+
+@app.route('/todos', methods=['POST'])
+def create_todo():
+    data = request.get_json()
+    print(data)
+    todo = repository.create_todo(data['task'], data['description'],data['completed'], data['edited'],data['createdDate'], data['modifiedDate'],data['lastViewed'],data['serialNumber'])
```
Comment (daniel-statsig) [inline pythontodo/src/todo_controller.py:17]: Cam we just pass `data` into the create_todo method?

File: pythontodo/src/todo_controller.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    print(todo.getJson())
```
Comment (daniel-statsig) [inline pythontodo/src/todo_controller.py:18]: remove print

File: pythontodo/main.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+from statsig import statsig
+
+import requests
+
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+
+
+# Make the request with SSL certificate verification enabled
+requests.get('https://statsigapi.net/v1/download_config_specs', verify=False)
```
Comment (daniel-statsig) [inline pythontodo/main.py:10]: What is the purpose of this? We should not need to directly hit statsigapi.net, the SDK should do all this for you.

File: pythontodo/src/todo.py (hunk 1 above)
Comment (upendras-ltts) [reply pythontodo/src/todo.py:34]: This we just added to match with Node server, for next server language, I will try to store it locally without database.

File: pythontodo/src/todo_controller.py (hunk 2 above)
Comment (upendras-ltts) [reply pythontodo/src/todo_controller.py:17]: I will try if that work, not much familiar with the Python, just used it as per some documentation.

File: pythontodo/src/todo_controller.py (hunk 3 above)
Comment (upendras-ltts) [reply pythontodo/src/todo_controller.py:18]: Sure

File: pythontodo/main.py (hunk 4 above)
Comment (upendras-ltts) [reply pythontodo/main.py:10]: I was not able to call the Statsig server due to some firewall issue, it fixed now, I will remove this line.

**Pull request merged**